src/pages/Pattern/CreatePattern.tsx

```diff
@@ -286,20 +304,26 @@ const CreatePattern = () => {
           <>
             <section className={styles.container}>
               <div className={styles.header}>
-                <StyledMUITextField
+                <Input
+                  status={helperTexts.name !== "" ? "error" : undefined}
+                  type="text"
                   value={name}
-                  label="Name"
+                  size="large"
+                  placeholder="Name"
                   onChange={(e: any) => setName(e.target.value)}
                 />
-                <StyledMUITextField
-                  value={durationInMinutes}
+
+                <Input
                   type="number"
-                  label="Duration (in Minutes)"
+                  status={helperTexts.durationInMinutes !== "" ? "error" : ""}
+                  value={durationInMinutes}
+                  placeholder="Duration (in Minutes)"
+                  size="large"
                   onChange={(e: any) => setDurationInMinutes(e.target.value)}
                 />
-                <MUISimpleAutocomplete
-                  label="Exam"
-                  onChange={setExam}
+                <AutoComplete
```

Comment:
Don't use Auto complete directly from AntD 
Check implementation in `CreateQuestion.tsx` We are using a custom component.